dist/svg/ds6/icons.svg

```diff
@@ -30,6 +30,12 @@
     <symbol id="icon-chevron-down-bold" viewBox="1.35 5.7 21.6 12.58">
         <path d="M12.186 18.285c-.451-.009-.809-.167-1.075-.441l-9.337-9.6a1.527 1.527 0 0 1-.424-.999v-.108c.015-.386.166-.741.424-1.008.56-.573 1.529-.57 2.082 0l8.294 8.53 8.292-8.532c.558-.57 1.526-.57 2.08 0 .265.27.416.629.428 1.01v.087c-.012.391-.165.75-.427 1.02l-9.333 9.6a1.443 1.443 0 0 1-1.004.441"/>
     </symbol>
+    <symbol id="icon-chevron-down-bold-blue" viewBox="1.35 5.7 21.6 12.58">
```

Comment:
So for background icons we do need to base64 encode a separate blue version (like you have done), but for foreground svgs we usually try to avoid putting a fill in the path. Instead we can set the color via CSS (or have it inherit. In that case we would just leverage the existing chevron symbols that don't have a fill. There are some exceptional cases where we have a fill in the symbol, but I'm not sure if this needs to be one of them.

Reply:
ok so we don't need symbols for these? @seangates

Reply:
Oh! Ian is right! I had completely forgot about the fill not being needed except for certain icons that never change color.

Thanks for catching my mistake, Ian!